frontend/tests/components/group/group-management-droplet-list.test.tsx

```
@@ -7,34 +7,7 @@ import {
-  }),
-  DndProvider: ({ children }: { children: React.ReactNode }) => children,
-}));
-
-jest.mock("react-dnd-html5-backend", () => ({
-  HTML5Backend: {},
-}));
-
-const renderWithDnd = (ui: React.ReactElement) => {
-  return render(<DndProvider backend={HTML5Backend}>{ui}</DndProvider>);
-};
+import { render, screen, fireEvent } from "@testing-library/react";
```

ESLint

[error] 'fireEvent' is defined but never used.

(@typescript-eslint/no-unused-vars)